pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -1076,6 +1076,17 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 				isVirtioNicRequested = true
 			}
 
+			if iface.DHCPOptions != nil {
+				for index, ip := range iface.DHCPOptions.NTPServers {
+					if net.ParseIP(ip).To4() == nil {
```

Comment:
The wire format specified in the DHCP protocol mandates exactly 4 octets per server address for option 42 (RFC 1533, section 8.3), so I hope this code will never be exercised outside of unit tests. :)

Reply:
Yeah, this check enforces that it is a valid IPv4 address.